src/runtime/runtime.ts

```diff
@@ -144,6 +178,12 @@ class Duomo implements Runtime {
 		return this.#darkMode
 	}
 	setDarkMode(mode: boolean) {
+		const durValue = getComputedStyle(document.documentElement).getPropertyValue('--default-theme-transition-duration');
```

Comment:
This looks good but we don’t need to compute `durValue` if we end returning early because of `this.#shouldNoOpDarkMode`. So I would move this somewhere under the `if (this.#shouldNoOpDarkMode) { ... }` code.